src/bq_data_collector.py

```diff
@@ -19,31 +18,32 @@
 
 
 class BigQueryDataCollector:
-    def __init__(self, repos: List[str], bq_credentials_path: str = '', days: int = 3):
+    def __init__(self, ecosystems: List[str], bq_credentials_path: str = '', days: int = 3):
         self._bq_client = BigQueryDataCollector._get_bq_client(bq_credentials_path)
-        self._repo_list = bq_client_helper.get_eco_system_with_repo_list()
-        self._init_query_param(repos, days)
+        self._repo_list = bq_client_helper.get_eco_system_with_repo_list(cc.REPO_LIST)
```

Comment:
Repo list must be argumented to BigQueryDataCollector constructor similar to bq_credentials_path.

Reply:
Now taking it as param